plugin/bullets.vim

```diff
@@ -558,10 +558,8 @@ endfun
 fun! s:set_checkbox(lnum, marker)
   let l:initpos = getpos('.')
   let l:pos = s:find_checkbox_position(a:lnum)
-  let l:checkbox_content = getline(a:lnum)[l:pos]
```

Comment:
checkbox_content appears to be unused, I removed it

Reply:
Good catch! I'm surprised the linter (Vint) didn't catch that. It runs on builds.